src/components/FieldNote/FieldNote.tsx

```diff
@@ -60,7 +60,7 @@ export const FieldNote = ({
           className={styles['field-note__icon']}
           title="error"
           purpose="informative"
-          name={isError && 'cancel'}
+          name={isError ? 'cancel' : undefined}
```

Comment:
I guess previously this would pass an empty string, which typescript didn't like